build/azure-pipelines/config/tsaoptions.json

```diff
@@ -1,16 +1,19 @@
 {
-	"codebaseName": "devdiv_vscode-client",
+	"codebaseName": "Azure Data Studio",
 	"ppe": false,
 	"notificationAliases": [
-			"sbatten@microsoft.com"
+		"kvhdir@microsoft.com",
+		"sakshis@microsoft.com"
 	],
 	"codebaseAdmins": [
-			"REDMOND\\stbatt",
-			"REDMOND\\monacotools"
+		"REDMOND\\karlb",
+		"REDMOND\\chgagnon",
+		"REDMOND\\kisantia",
+		"REDMOND\\sakshis"
 	],
-	"instanceUrl": "https://devdiv.visualstudio.com/defaultcollection",
-	"projectName": "DevDiv",
-	"areaPath": "DevDiv\\VS Code (compliance tracking only)\\Visual Studio Code Client",
+	"instanceUrl": "https://msdata.visualstudio.com",
+	"projectName": "Database Systems",
+	"areaPath": "Database Systems\\SQL Tools\\Azure Data Studio",
 	"notifyAlways": true,
 	"template": "TFSDEVDIV",
```

Comment:
Do you know what this template refers to? Do we still want to use this one?

Reply:
I am not sure what it refers to. I can remove it in another PR, just to make sure I don't mess this up in case it is indeed needed (and make sure that the TSA report indeed gets generated).